### PR TITLE
Add book list per category

### PR DESCRIPTION
**Pull request by leila100**
When clicking on a category, it will display a list of books from that category
Each book title is a link to a more detailed page about that book (the page is empty for now)

### CODE REVIEW

**Review by johnoro — COMMENTED** [review]
It lives! The continuous integration is great to know everything's good in terms of deployment.

**Comment by leila100** [comment]
Yes it worked! 😊

Sent from Mail for Windows 10

File: oer-bookr/src/components/book/Book.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        </div>
+        <div>
+          <span>Publisher:</span> {props.book.publisher}
+        </div>
+        <div>
+          <span>License:</span> {props.book.license}
+        </div>
+        <div>
+          <span>Reviews:</span> {props.book.reviews}
+        </div>
```
Comment (johnoro) [inline oer-bookr/src/components/book/Book.js:61]: It feels like some of this code could be more programmatic; for example:
```javascript
// ...
const capitalize = str => str[0].toUpperCase() + str.slice(1)
const sections = [ 'author', 'publisher', 'license', 'reviews' ]
const makeSectionDiv = name => (
  <div>
    <span>{capitalize(section)}:</span> {props.book[section]}
  </div>
)

return  (
  <BookWrapper>
    <img /* ... */ />
    <BookInfo>
      <div>
        <span>Title:</span>
        <Link to={`/book/${props.book.id}`}> {props.book.title}</Link>
      </div>
      {sections.map(makeSectionDiv)}
    </BookInfo>
  </BookWrapper>
)
// ...
```
It also feels like the switch statement could be made simpler somehow in a similar sense. I don't have an immediate way of doing it coming to my head, though! I think it'd involve an object containing the images.

Comment (leila100) [reply oer-bookr/src/components/book/Book.js:61]: Good idea for showing the sections of the book info. For the switch statement, it's just dummy data for now. I will be getting the information/images from backend (I think)

Comment (leila100) [reply oer-bookr/src/components/book/Book.js:61]: Not sure how to display images otherwise. In React you have to import the images unless you have a url

Comment (johnoro) [reply oer-bookr/src/components/book/Book.js:61]: Yeah, it'd be importing and then storing them in an object after the fact, which would be adding an extra step in a sense, and then you could access them via their keys (the same as the variable names that you imported them as). 

You will be getting data from the back-end later, so I definitely agree there!

**Pull request merged**